-Test 1: Teste 1 (-20.000)
Incorrect program result
 --- Input ---
> 3
>4
>0
>0
>0
>1
>0
>0
>1
>1
>0
>0
>0
>0

 --- Program output ---
>linhas:colunasTraceback (most recent call last):
>  File "matriz1.py", line 41, in <module>
>    for i in range(0,x.shape[0],1):
>NameError: name 'x' is not defined

 --- Expected output (text)---
> '[[ 0.  1.]
> [ 1.  1.]]'
>
>

-Summary of tests
>+------------------------------+
>|  1  test run/ 0 tests passed |
>+------------------------------+

